Cartridges/bm_checkoutcom/cartridge/scripts/helpers/CKOHelper.js

```diff
@@ -312,8 +315,31 @@ var CKOHelper = {
      * @param {number} amount The amount to format
      * @returns {number} The formatted amount
      */
-    getFormattedPrice: function(amount) {
-        return amount * 100;
+    getFormattedPrice: function(amount, currency) {
+        var totalFormated;
+        if (currency) {
+            var ckoFormateBy = this.getCkoFormatedValue(currency);
```

Comment:
I think you can make ckoFormateBy = currency ? this.getCkoFormatedValue(currency) : 100; and remove the if else completely

Reply:
Good feedback, but after looking at the code, we don't want to return 100. Return would be 100 * amount and toFixed().